docs/appendices/release-notes/5.8.5.rst

```diff
@@ -40,6 +40,11 @@ See the :ref:`version_5.8.0` release notes for a full list of changes in the
 Fixes
 =====
 
+- Fixed an issue that caused wrong results when implicit joins were combined
+  with outer-joins within the same query e.g.::
+
+      SELECT * FROM t1, t2 RIGHT JOIN t3 ON true
+
 - Fixed an issue which would allow to cast a number to a
```

Comment:
5.8.5 has been released, we don't do 5.8.* releases anymore, can be removed

Reply:
👍